### PR TITLE
boards/nucleo-l053: initial support

### PR DESCRIPTION
**Pull request by aabadie**
This adds basic support for [Nucleo-L053R8](http://www.st.com/en/evaluation-tools/nucleo-l053r8.html) and is based on #6433.

### CODE REVIEW

**Comment by aabadie** [comment]
Needs #6433

**Comment by lebrush** [comment]
I've a L053 laying around, I'll give it a try today.

**Comment by lebrush** [comment]
As expected, same situation as for the L073 :-)

 - hello_world **OK**
 - ipc_pingpong **OK**
 - timer_periodic_wakeup **HANGS**

**Comment by aabadie** [comment]
Fixed the xtimer here as well. All is working good.

File: boards/nucleo-l053/include/board.h [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+ * @defgroup    boards_nucleo-l053 Nucleo-L053
+ * @ingroup     boards
+ * @brief       Board specific files for the nucleo-l053 board
+ * @{
+ *
+ * @file
+ * @brief       Board specific definitions for the nucleo-l053 board
+ *
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ * @author      Alexandre Aabdie <alexandre.abadie@inria.fr>
```
Comment (lebrush) [inline boards/nucleo-l053/include/board.h]: @aabadie typo on your surname

File: boards/nucleo-l053/include/periph_conf.h [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+
+/**
+ * @ingroup     boards_nucleo-l053
+ * @{
+ *
+ * @file
+ * @brief       Peripheral MCU configuration for the nucleo-l053 board
+ *
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ * @author      Alexandre Aabdie <alexandre.abadie@inria.fr>
```
Comment (lebrush) [inline boards/nucleo-l053/include/periph_conf.h]: same here

File: boards/nucleo-l053/include/board.h (hunk 1 above)
Comment (aabadie) [inline boards/nucleo-l053/include/board.h]: oh! good catch :)

Comment (aabadie) [inline boards/nucleo-l053/include/board.h]: update force-pushed

**Comment by aabadie** [comment]
looks like the CI are having issues ? @kaspar030, @cgundogan any idea ?

**Pull request merged**